src/BuildScriptGeneratorCli/Commands/CommandBase.cs

```diff
@@ -24,7 +24,7 @@ internal abstract class CommandBase
         public string LogFilePath { get; set; }
 
         [Option("--debug", Description = "Print stack traces for exceptions.")]
```

Comment:
I am not suggesting to make a change here, but I think this should probably be `--debugInfo`